src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  onClearCompleted: () => void;
+  onFilterChange: (filter: Filter) => void;
+  activeFilter: Filter;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onClearCompleted,
+  onFilterChange,
+  activeFilter,
+}) => {
+  const handleClearCompleted = () => {
```

Comment:
You can directly use 'onClearCompleted' instead of creating a new function 'handleClearCompleted'.